libs/testdiff/testdiff.go

```diff
@@ -0,0 +1,91 @@
+package testdiff
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/databricks/cli/internal/testutil"
+	"github.com/hexops/gotextdiff"
+	"github.com/hexops/gotextdiff/myers"
+	"github.com/hexops/gotextdiff/span"
+	"github.com/stretchr/testify/assert"
+	"github.com/wI2L/jsondiff"
+)
+
+func UnifiedDiff(filename1, filename2, s1, s2 string) string {
+	edits := myers.ComputeEdits(span.URIFromPath(filename1), s1, s2)
+	return fmt.Sprint(gotextdiff.ToUnified(filename1, filename2, s1, edits))
+}
+
+func AssertEqualTexts(t testutil.TestingT, filename1, filename2, expected, out string) {
+	if len(out) < 1000 && len(expected) < 1000 {
```

Comment:
1000 is just arbitrary number here, right?

Reply:
yes

Reply:
Constants (used in multiple spots) can be a top level `const`, e.g. `textDiffCharacterThreshold` or smth.

Reply:
In general agree, but I think here it's easier to read if it's all in one place. We can refactor later if that causes confusion.